messages/ruby/lib/cucumber/messages.dtos.rb

```diff
@@ -588,6 +604,16 @@ def initialize(
         @children = children
         @id = id
       end
+
+      private
```

Comment:
just having one private and then a line space is enough (Again aware this is auto-genned).

Reply:
Yes I know
Managing that in the ruby codegen erb template would add a lot of noise
I am not sure the benefit of having a proper automaticaly generated file would override that cost